src/swift/flwr/Sources/Flower/CoreML/MLFlwrClient.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  File.swift
+//  
+//
+//  Created by Daniel Nugraha on 18.01.23.
+//
+
+import Foundation
+import NIOCore
+import NIOPosix
+import CoreML
+
+public enum MLTask {
+    case train
+    case test
+}
+
+@available(iOS 14.0, *)
+public class MLFlwrClient: Client {
+    private var eventLoopGroup: EventLoopGroup?
+    
+    private var parameters: MLParameter
+    private var dataLoader: MLDataLoader
+    
+    private var compiledModelUrl: URL
+    private var tempModelUrl: URL
+    
```

Comment:
Can you add some documentation regarding the purpose of this class?

Reply:
I am no Swift expert and I am wondering if this should be rather user-level code but I am not sure. Could you explain a bit about the reasoning this should go into the flwr core and not be part of the example?

Reply:
The idea is to provide one Client as a default, just like numpy Client in Python. Do you think it is more appropriate to move it to example?

Reply:
@danielnugraha, that depends on how well it generalizes. NumpyClient mostly takes care of the conversion to ensure no one has to be an expert in protobuf. Most researchers are pretty familiar with Numpy. Do you think this reasoning applies here, too? If so, we can keep it and improve on it later. I am looking forward to hearing your thoughts.